homeassistant/components/nest/climate_sdm.py

```diff
@@ -290,7 +290,9 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
         try:
             await trait.set_mode(api_mode)
         except ApiException as err:
-            raise HomeAssistantError(f"Error setting HVAC mode: {err}") from err
+            raise HomeAssistantError(
+                f"Error setting HVAC mode {self.entity_id} to {hvac_mode}: {err}"
```

Comment:
What do you think about this word ordering? This is what reads more natural to me:

```suggestion
                f"Error setting {self.entity_id} HVAC mode to {hvac_mode}: {err}"
```

Reply:
I like your suggestion, I'll make those changes.

Reply:
Updated all four of them, adjusted unit tests